frontend/src/v5/ui/routes/userSignup/userSignupForm/userSignupFormStep/userSignupFormStepAccount/userSignupFormStepAccount.component.tsx

```diff
@@ -108,7 +109,19 @@ export const UserSignupFormStepAccount = ({
 
 	return (
 		<>
-			<FormTextField
+			<ControlledInput
```

Comment:
This loses focus when an error is triggered.
So you can't type a username without it losing focus twice as the username needs to be 2 characters long

Reply:
The same happens with other fields in this step. It doesn't seem to be an issue with the fields in step 2